src/wp-admin/includes/class-wp-upgrader.php

```diff
@@ -710,6 +710,20 @@ public function install_package( $args = array() ) {
 		if ( is_wp_error( $res ) ) {
 			$this->result = $res;
 			return $res;
+		} else {
+			$dependencies = false;
+			if ( ! empty( $this->new_plugin_data['RequiresPlugins'] ) ) {
+				$dependencies = $this->new_plugin_data['RequiresPlugins'];
+				$dependencies = WP_Plugin_Dependencies::sanitize_dependency_slugs( $dependencies );
+			}
+
+			update_site_option(
+				'wp_plugin_dependencies_known_dependencies',
+				array_merge(
+					get_site_option( 'wp_plugin_dependencies_known_dependencies', array() ),
+					array( $this->new_plugin_data['Slug'] => $dependencies )
+				)
+			);
```

Comment:
I should probably only be running this during a _plugin_ package install. 😅 Will update this later.

Reply:
Done 👍